models/engine/db_storage.py

```diff
@@ -74,3 +74,18 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """ Method to return objects from given cls, id """
+        dic = self.all(cls)
+        key = str(cls.__name__) + "." + id
+        if (key in dic.keys()):
```

Comment:
The parenthesis are not needed in this if statement. However, you can keep it as it is.